projects/vdk-plugins/vdk-oracle/tests/jobs/oracle-ingest-job-case-insensitive-neg1/20_ingest.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023-2024 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+import datetime
+
+
+def run(job_input):
+    payloads = [
+        # Schema wil be inferred but nothing will be ingested
```

Comment:
Why is it inferred? If nothing is going to be ingested can't we just "ignore" the whole operation